src/main/java/org/terasology/spawning/SpawnerSystem.java

```diff
@@ -171,7 +181,7 @@ public void onSpawn(PeriodicActionTriggeredEvent event, EntityRef spawner) {
      */
     // TODO: Switch away from UpdateSubScriberSystem to the DelaySystem and handle tick "throttle" there. Then simplify
     public void update(float delta) {
-        // Do a time check to see if we should even bother calculating stuff (really only needed every second or so)
+/*        // Do a time check to see if we should even bother calculating stuff (really only needed every second or so)
```

Comment:
Removed the the code which was running the update function every second, so that it is run every tick, as soon as the user places portal and gives the required resources their Oreon should be spawned, so favoring update every tick